test/reuse/nonUi5/element/getById.spec.js

```diff
@@ -32,4 +32,21 @@ describe("element - getById and catch error", function () {
     await expect(nonUi5.element.getById("sdk---app--apiTab-text"))
       .rejects.toThrow("Function 'getById' failed");
   });
+});
+
+describe("element - getById - error case, verify error stack", function () {
+
+  it("Preparation", async function () {
+    await common.navigation.navigateToUrl("https://sapui5.hana.ondemand.com/1.99.0/");
+    await handleCookiesConsent();
+  });
+
+  it("Execution & Verification", async function () {
+    try {
+      await nonUi5.element.getById("sdk---app--apiTab-text");
```

Comment:
can we change to expectToThrow or sth. similar?

Reply:
@Templada - I couldn't find anything that would match the error stack. `rejects.toMatch`, `rejects.toThrow` and related methods only match the error message, not the stack trace.